add Community helps to getting started page and add 2 solid apps to apps page

soltanireza65 opened the pull request.

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
 Once you are set up with your own Pod and WebID,
 you're ready to start [developing your own app](/developers/tutorials/first-app).
+
+## Community helps
```
```suggestion
## Community assistance
```

soltanireza65 replied:
fixed

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
+
+## Community helps
+
+Check some of the applications. Some interesting approaches
```
```suggestion
Check some of the applications. Some interesting approaches follow:
```

soltanireza65 replied:
appliyed

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
+
+Check some of the applications. Some interesting approaches
+
+- PodOS https://github.com/pod-os/PodOS (Web Components)
+- solid-uix https://github.com/SolidOS/solid-uix (Native Web Components)
```
```suggestion
- **PodOS** (Web Components) https://github.com/pod-os/PodOS
- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix
```

soltanireza65 replied:
appliyed

soltanireza65 commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -34,3 +34,19 @@ to run on your own machine.
+- and also this one [solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)
+
+Cou can also checkout the solid apps at:
+- [Solid Apps](https://solidproject.org/apps)
```
appliyed

soltanireza65 commented:
@TallTed 
I've just added a new section "Youtube videos"

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,20 +36,23 @@ For instructions on:
 
 Once you are set up with your own Pod and WebID,
-you're ready to start [developing your own app](/developers/tutorials/first-app).
+you're ready to develop your own app. Various Solid client libraries may provide a beginning app tutorial; for example, the [Inrupt JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```
```suggestion
you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,20 +36,23 @@ For instructions on:
+## Community assistance
 
-Check some of the applications. Some interesting approaches
+Check some of the applications. Some interesting approaches follow:
```
```suggestion
Check out some of the existing applications. Some interesting approaches follow:
```

TallTed commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,20 +36,23 @@ For instructions on:
+- [Solid Apps](https://solidproject.org/apps)
+
+Youtube videos:
+- [How we're doing Solid - In less than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)
```
```suggestion
- [How we're doing Solid — in fewer than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)
```

TallTed reviewed (changes requested):
Small stuff

soltanireza65 commented:
> Small stuff

fixed

michielbdejong commented:
@TallTed can you re-review please?

TallTed reviewed (changes requested):
Different small stuff :-)

soltanireza65 commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,4 +36,23 @@ For instructions on:
 
 Once you are set up with your own Pod and WebID,
-you're ready to develop your own app. Various Solid client libraries may provide a beginning app tutorial; for example, the [Inrupt JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
+you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```
applied

soltanireza65 commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,4 +36,23 @@ For instructions on:
+- **PodOS** (Web Components) https://github.com/pod-os/PodOS
+- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix
+
+Solid With web frameworks:
```
applied

soltanireza65 commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,4 +36,23 @@ For instructions on:
+
+Solid With web frameworks:
+
+- **ember-solid**: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden)
```
applied

soltanireza65 commented on _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:
```diff
@@ -36,4 +36,23 @@ For instructions on:
+
+- **ember-solid**: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden)
+- **[todo-ldo](https://github.com/megoth/todo-ldo)** a [solid/nextjs/ts] app (Arne Hassel)
+- **[solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)**
```
applied

soltanireza65 commented:
@TallTed 
I've just applied the requested changes,
As part of the last 2 commits, I've just added my new 2 solid apps to the apps page.
that's why I've just updated the PR title.

TallTed commented:
@soltanireza65 — It is often better to use the "Commit suggestion" or "Add suggestion to batch" buttons, as these both apply the suggested change *and* resolve the conversation without requiring manual review of manually applied changes (which I'm doing now)....
